awsutil/route53.go

```diff
@@ -182,7 +183,16 @@ func (r *Route53) DescribeResourceRecordSets(zoneID *string, hostname *string) (
 		return nil, fmt.Errorf("ListResourceRecordSets(%s, %s) returned an empty list", *zoneID, *hostname)
 	}
 
-	return resp.ResourceRecordSets[0], nil
+	for _, record := range resp.ResourceRecordSets {
+		if *record.Type != "CNAME" && *record.Type != "A" {
+			continue
```

Comment:
For consistency use constants from https://github.com/aws/aws-sdk-go/blob/master/service/route53/api.go#L13538-L13571